lib/rules/block-opening-brace-space-after/__tests__/index.js

```diff
@@ -467,3 +467,29 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: ['always', { ignore: ['at-rules'] }],
+	fix: true,
+	accept: [
+		{
+			code: 'a { color: pink; }',
+		},
+		{
+			code: '@media print { a { color: pink; } }',
+		},
+		{
+			code: '@media print {\na { color: pink; } }',
+		},
+	],
+	reject: [
+		{
+			code: 'a {color: pink; }',
+			fixed: 'a { color: pink; }',
+			message: messages.expectedAfter(),
+			line: 1,
+			column: 4,
```

Comment:
```suggestion
			code: '@media print {\n  a {color: pink; }',
			fixed: '@media print {\n  a { color: pink; }',
			message: messages.expectedAfter(),
			line: 1,
			column: 21,
```

Let's check that only the rule is fixed.